src/components/Home.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import './Calculator.css';
+
+class Home extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div>
+        <h2 className="Heading-1"> Welcome to our page! </h2>
+
+        <p className="Description">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam sed
+          vestibulum erat. Nulla sagittis massa at egestas facilisis. Quisque
+          tincidunt interdum vulputate. Sed orci orci, viverra vel tincidunt vel,
+          consectetur eget tortor. Integer pellentesque eros vel ex maximus, quis
+          aliquet nisl tristique. Nulla lobortis sem a placerat maximus. Duis
+          suscipit purus at nunc bibendum, et finibus quam tempor. Donec sed magna
+          suscipit, bibendum nisi faucibus, dictum erat. Aenean varius eros vitae
+          arcu tempus lobortis. Ut et nisl eget purus ornare tincidunt. Aliquam erat
+          volutpat. Suspendisse vitae finibus odio. Suspendisse aliquam risus massa,
+          vel vulputate mauris tempus a. In ullamcorper arcu eu suscipit facilisis.
+          Suspendisse at elit et arcu finibus pulvinar. Fusce at ullamcorper libero.
+          Nunc eget diam rutrum, lobortis sem vel, cursus lacus. Aliquam at elit nec
+          est ultrices pharetra sit amet et dolor. Pellentesque mattis sodales
+          libero, suscipit ullamcorper neque. Duis feugiat maximus faucibus.
+          Praesent consectetur fermentum velit non eleifend. Cras nec lorem
+          interdum, aliquam nulla sed, elementum arcu. Vivamus non maximus nisi.
+          Duis quis lacinia sem, nec imperdiet sapien. Vivamus a ipsum ut nibh
+          congue venenatis.
+
+        </p>
+      </div>
+    );
+  }
+}
+export default Home;
```

Comment:
[REQUIRED] Kindly align the texts. more to the left so that the match the general layout of the wireframe

![image](https://user-images.githubusercontent.com/62938735/213865456-afc0158f-2bd6-4f9b-bac8-d66b848b71f6.png)